mkdocs.yml

```diff
@@ -11,6 +11,7 @@ nav:
   - Home: index.md
   - Usage and Configuration: usage.md
   - Issues detection: issues-detection.md
+  - Changelog: CHANGELOG.md
```

Comment:
`mkdocs` doesn't allow linking to files outside of `docs`, so we need to create a symlink to it.